### PR TITLE
Gamma source generator

### PR DESCRIPTION
**Pull request by octagonz82**
New function for MCNP2ACAB, it creates SDEF definition for multiple cells and times

### CODE REVIEW

File: src/mc2acab/MCNP_ACAB.py [hunk 1]
```diff
@@ -86,11 +87,22 @@ def MCNP_ACAB_Mapstar(n):
...
     '-decay_outs' : None,
+    '-sdef' : None,
     '-passive_sector' : None, # so far useless
-    '-nuc_lib': 'EAF', 
-    '-id_Egroup': 'vitJ+', 
+    '-nuc_lib': 'EAF',
+    '-id_Egroup': 'vitJ+',
 }
 def __parse_args(reqs, options, args):
+    if os.path.isfile('logfile.txt'):
```
Comment (MigMagan) [inline src/mc2acab/MCNP_ACAB.py]: This is not related to the PR topic, remove and we can add it in another commit.

File: src/mc2acab/MCNP_ACAB.py [hunk 2]
```diff
@@ -155,6 +169,9 @@ def __parse_args(reqs, options, args):
                                   ' is not here \nEnter outp file name: ')
     if reqs['-st_units'] == '2':
         reqs['-st'] *= 6.24E15
+    if options['-sce_file']:
```
Comment (MigMagan) [inline src/mc2acab/MCNP_ACAB.py]: Remove from this PR and add it in another commit.

**Review by MigMagan — CHANGES_REQUESTED** [review]
Revert the 2 changes, copying the code for another commit in master.

File: src/mc2acab/MCNP_ACAB.py (hunk 2 above)
Comment (octagonz82) [reply src/mc2acab/MCNP_ACAB.py]: OK

File: src/mc2acab/MCNP_ACAB.py (hunk 1 above)
Comment (octagonz82) [reply src/mc2acab/MCNP_ACAB.py]: OK

**Review by MigMagan — APPROVED** [review]
All clear!
